encrypted-media/scripts/keystatuses.js

```diff
@@ -49,10 +49,11 @@ function runTest(config,qualifier)
             }
             function lexicographical( a, b ) { return a < b ? -1 : a === b ? 0 : +1; }
             function lexicographicalkey( a, b ) { return lexicographical( a.key, b.key ); }
-            var expected = [{ key: key1String, value: 'usable'}, { key: key2String, value: 'usable'}].sort( lexicographicalkey );
-            assert_equals(JSON.stringify(result),
-                          JSON.stringify(expected),
-                                 "keystatuses should have the two expected keys with keystatus 'usable'");
+            var expected1 = [{ key: key1String, value: 'usable'}, { key: key2String, value: 'usable'}].sort( lexicographicalkey );
+            var expected2 = [{ key: key1String, value: 'status-pending'}, { key: key2String, value: 'status-pending'}].sort( lexicographicalkey );
+            assert_in_array(    JSON.stringify(result),
+                                [ JSON.stringify(expected1),JSON.stringify(expected2) ],
+                                 "keystatuses should have the two expected keys with keystatus 'usable' or 'status-pending'");
```

Comment:
Is there a way to understand if status should be exactly 'usable' or exactly 'status-pending'?